partner_contact_sale_info_propagation/models/res_partner.py

```diff
@@ -26,22 +25,23 @@ def write(self, vals):
                 )
                 if childs:
                     childs.write({"team_id": vals["team_id"]})
-        return super(ResPartner, self).write(vals)
+        return super().write(vals)
 
-    @api.model
-    def create(self, vals):
-        if "parent_id" in vals:
-            if "user_id" not in vals:
-                vals.update(user_id=self.browse(vals["parent_id"]).user_id.id)
-            if "team_id" not in vals:
-                vals.update(team_id=self.browse(vals["parent_id"]).team_id.id)
-        return super(ResPartner, self).create(vals)
+    @api.model_create_multi
+    def create(self, vals_list):
+        for vals in vals_list:
+            if vals.get("parent_id"):
```

Comment:
This is avoiding to propagate/assign false values, so please don't change the previous syntax